beacon-chain/core/electra/upgrade.go

```diff
@@ -268,9 +261,9 @@ func UpgradeToElectra(beaconState state.BeaconState) (state.BeaconState, error)
 
 		DepositRequestsStartIndex:     params.BeaconConfig().UnsetDepositRequestsStartIndex,
 		DepositBalanceToConsume:       0,
-		ExitBalanceToConsume:          helpers.ActivationExitChurnLimit(primitives.Gwei(tab)),
+		ExitBalanceToConsume:          0,
```

Comment:
I dont think this is right either. We should assign them to churn limit based on total active balances
```
    post.exit_balance_to_consume = get_activation_exit_churn_limit(post)
    post.consolidation_balance_to_consume = get_consolidation_churn_limit(post)
```

Reply:
Ugh. Once again you're right. I'm going to close this.